### PR TITLE
polok docker task 2

### PR DESCRIPTION
**Pull request by codefresher32**
Fixes #

- [ ] Does not depend on other PRs
- [x] Depends on other PR being merged first: #300 

## What kind of change does this PR introduce (check at least one)

- [ ] Bugfix
- [ ] Improvement
- [x] New feature
- [ ] Refactor
- [ ] Infrastructure
- [ ] Other (please specify)

## How would you describe the changes in a changelog (1-2 lines)?
- Running todo app both in normul build and production build

## What was done
- Added compose yml both for normul & production build
- Added diiferent dockerfile for normul & production build

## How to test

- [ ] Needs regression testing (add QA needed label)

## Deploy risk & follow up

Please assess risk & follow up on deployment of this PR. Elaborate your choice if selecting anything other than 'No risk'.

- [x] No risk
- [ ] Low
- [ ] High

## Reminder list

- Determine whether REST API Documentation should be updated
- Make an assessment on whether unit tests should be updated and/or added
- Ensure follow up tasks are linked to this PR

QA - if regression testing was requested:

- [ ] QA has verified

### CODE REVIEW

File: react_client/todo/Dockerfile.prod [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+#  - using node becuase it is the required runtime 
+FROM node:alpine AS builder
+
+# Defining a workir so our apllication code is in a specific folder for organizing our container
+WORKDIR /app
+
+# Copying package.json as it includes project defination our our react project and is required to run the project
+# Only copying this file specifily 
+#   because this file is the least likely to change
+COPY package.json .
```
Comment (musfique-m) [inline react_client/todo/Dockerfile.prod:11]: Please also copy `package-lock.json` to keep track of the exact version of every package that is installed.
Also use wild card to reduce the number of layers.

**Review by musfique-m — CHANGES_REQUESTED** [review]

File: react_client/todo/Dockerfile.prod (hunk 1 above)
Comment (codefresher32) [reply react_client/todo/Dockerfile.prod]: Added package-lock.json as wildcard....

**Comment by codefresher32** [comment]
Reverted the commit. Please review this pr.

**Review by musfique-m — APPROVED** [review]
Looks good.

**Pull request merged**